app/views/spree/admin/variants/_autocomplete.js.erb

```diff
@@ -57,7 +57,7 @@
                 <button class="add_variant no-text icon-plus icon_link with-tip" data-stock-location-id="{{variant.stock_location_id}}" title="Add" data-action="add"></button>
               </td>
             {{else}}
-              <td><%= Spree.t(:out_of_stock) %></td>
+              <td><%= t('admin.subscriptions.stock.out_of_stock') %></td>
```

Comment:
We try not to reuse translation keys, we add new ones in these cases so that translators have the flexibility later.

You can add this one as a lazy lookup like t('.out_of_stock')
I think that will fit in line 2666:
![image](https://user-images.githubusercontent.com/1640378/82660065-97089f80-9c21-11ea-9aeb-9d6698f85662.png)

Please test it, I am not sure this location is correct.
One easy way to test translations locally is to change the translation to something different and see if it is correctly picked up like:
out_fo_stock: "Out of Stock TEST"

Reply:
Changes in, fingers crossed ;-)

Reply:
looks good, when you say fingers crossed it makes me wonder if you have managed to test this locally. Were you able to test this one locally? If not, let me know I can help you with that.

Reply:
I did test locally and all looked good to me. It's more of "what might the reviewers find that I missed?" Love learning and all that goes with it :-)

Reply:
awesome, thanks!